src/actions/guides/database/models.cr

```diff
@@ -65,7 +65,19 @@ class Guides::Database::Models < GuideAction
     * `created_at` - default `Time` type.
     * `updated_at` - default `Time` type.
 
-    To skip adding these, call the `skip_default_columns` macro in your model or `BaseModel`.
+    To change your defaults, define a macro called `default_columns` in your model.
```

Comment:
You'd probably want this in your `BaseModel` by default so all your models get these